tools/mmp/mmp.csproj

```diff
@@ -5,47 +5,31 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{F3232882-0FA0-4BB6-9D9C-E2CC779EAF0D}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>linker</RootNamespace>
     <AssemblyName>mmp</AssemblyName>
+    <RootNamespace>mmp</RootNamespace>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <UseXamMacFullFramework>true</UseXamMacFullFramework>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>True</DebugSymbols>
-    <DebugType>full</DebugType>
+    <DebugType>portable</DebugType>
     <Optimize>False</Optimize>
     <OutputPath>bin\Debug</OutputPath>
-    <DefineConstants>MONOMAC;MMP;DEBUG</DefineConstants>
+    <DefineConstants>MONOMAC;MMP;XAMARIN_APPLETLS;DEBUG</DefineConstants>
```

Comment:
I thought we killed `XAMARIN_APPLETLS` ? it's the only option today, why is `mmp` need it ?

Reply:
I didn't check if it was needed, I just made sure the csproj does what the Makefile did.